com.unity.netcode.gameobjects/Runtime/Messaging/Messages/AnticipationTickSync.cs

```diff
@@ -0,0 +1,70 @@
+namespace Unity.Netcode
+{
+    internal struct AnticipationCounterSyncPingMessage : INetworkMessage, INetworkSerializeByMemcpy
```

Comment:
Why are there two serialize implementations for this message? INetworkMessage and INetworkSerializeByMemcpy

Reply:
`INetworkMessage` is not a serialization implementation, it is an interface to allow the messagin infrastructure to call `Serialize()` on it, which can then serialize in any way. With `INetworkSerializeByMemcpy`, it could then call `writer.WriteValueSafe(this)` to memcpy the whole thing in. So it's not unreasonable to have both.

But as I'm not doing that, `INetworkSerializeByMemcpy` is pretty useless and should be removed. It's a copy/paste error.